native-extension-examples/page-popup/README.md

```diff
@@ -0,0 +1,81 @@
+# Page Popup Native Extension
+
+## What is it?
+This Native Extension makes use of the angular material library to render a stylized dialog over the screen when the user navigates to the cart page. It can easily be adjusted to appear on any other page, and the same dialogs can be used to build more professional UI for native extension components. 
```

Comment:
```suggestion
This Native Extension uses the Angular Material library to render a stylized dialog over the screen when the user navigates to the cart page. It can easily be adjusted to appear on any other page, and the same dialog can be used to build more professional UI for native extension components. 
```